packages/react-component-library/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -16,11 +33,76 @@ const StyledContextMenu = styled.ol`
   box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.07);
 `
 
-export const ContextMenu: React.FC<ComponentWithClass> = ({
+function withinBoundingBox(
+  bottomLeft: Coordinate,
+  topRight: Coordinate,
+  point: Coordinate
+): boolean {
+  const isXInRange: boolean = point.x >= bottomLeft.x && point.x <= topRight.x
+  const isYiInRange: boolean = point.y <= bottomLeft.y && point.y >= topRight.y
+
+  return isXInRange && isYiInRange
+}
+
+export const ContextMenu: React.FC<ContextMenuProps> = ({
   className,
   children,
+  attachedToRef,
 }) => {
-  return <StyledContextMenu className={className}>{children}</StyledContextMenu>
+  const [isOpen, setIsOpen] = useState<boolean>(false)
+  const [position, setPosition] = useState<Coordinate>({ x: 0, y: 0 })
+
+  function displayContextMenu(e: MouseEvent) {
+    const {
+      top,
+      right,
+      bottom,
+      left,
+    } = attachedToRef.current.getBoundingClientRect()
+
+    const bottomLeft = { x: left, y: bottom }
+    const topRight = { x: right, y: top }
+    const mousePoint = { x: e.clientX, y: e.clientY }
+
+    setPosition(mousePoint)
+
+    if (withinBoundingBox(bottomLeft, topRight, mousePoint)) {
+      e.preventDefault()
+      setIsOpen(true)
+    } else {
+      setIsOpen(false)
```

Comment:
Is it possible to get a test to cover this line?

Reply:
Turns out these lines are being hit but Jest is getting confused with its coverage. 

Possibly because these functions are only invoked as event listeners?

Commenting out lines creates test failures.